Makefile-deps.mk

```diff
@@ -111,8 +111,12 @@ mpi-operator-crd: ## Copy the CRDs from the mpi-operator to the dep-crds directo
 KF_TRAINING_ROOT = $(shell $(GO_CMD) list -m -mod=readonly -f "{{.Dir}}" github.com/kubeflow/training-operator)
 .PHONY: kf-training-operator-crd
 kf-training-operator-crd: ## Copy the CRDs from the training-operator to the dep-crds directory.
+	## Full version of the manifest is required for e2e multikueue tests.
 	mkdir -p $(EXTERNAL_CRDS_DIR)/training-operator/
-	cp -f $(KF_TRAINING_ROOT)/manifests/base/crds/* $(EXTERNAL_CRDS_DIR)/training-operator/
+	cp -prf $(KF_TRAINING_ROOT)/manifests/* $(EXTERNAL_CRDS_DIR)/training-operator/
+	## Removing kubeflow.org_mpijobs.yaml is required as the version of MPIJob is conflicting between training-operator and mpi-operator - in integration tests.
```

Comment:
why is it only a problem for integration and not E2E?

Reply:
Let me explain.
**Integration tests** suit requires to specify the CRD directory and it doesn't care about the customise.yaml there, only plain CRDs - https://github.com/kubernetes-sigs/kueue/blob/afb49468223ca81419b13e45923f752f803dd80e/test/integration/multikueue/suite_test.go#L94
Thus we can't have MPIJob CRD from training-operator there, we need to physically remove the file from the folder.

**E2E** doesn't care about the MPIJob file, because we have our own kustomize in test/e2e/config/multikueue, where we patch it so MPIJob CRD is deleted.

As you can see we have different needs depending on the test, that can't be satisfied with the current approach.
Previously I had combined the two by using sed to remove MPIJob file entry from original kustomize and also delete the fie from the folder.